src/metabase/pulse/render/table.clj

```diff
@@ -116,7 +122,11 @@
                       (row-style-for-type cell)
                       {:background-color (get-background-color cell (get column-names col-idx) row-idx)}
                       (when (and bar-width (= col-idx 1))
-                        {:font-weight 700}))}
+                        {:font-weight 700})
+                      (when (= row-idx (dec (count rows)))
+                        {:border-bottom 0})
+                      (when (= col-idx (dec (count row)))
+                        {:border-right 0}))}
```

Comment:
These 2 conditions are needed because I change removed `border-collapse :collapse` to support rounded border (I couldn't figure out how to have rounded border with `border-collapse :collapse`)

That's why the right most column and the bottom most row shouldn't render their borders.

I did considered not rendering the right and bottom border of the table instead, but it seems it make the rounded corner looks jaggy.
#### Current implementation
![image](https://user-images.githubusercontent.com/1937582/190605063-beafc7d1-15b8-47cb-a2ff-e5a5dee83c87.png)
![image](https://user-images.githubusercontent.com/1937582/190605794-015edf38-a437-45c4-8f3c-22fd2757dbff.png)


VS
#### The implementation that doesn't render right and bottom border of the table instead
![image](https://user-images.githubusercontent.com/1937582/190605570-353f0aef-ed97-4afa-9e4a-e7a0b6a474a1.png)
![image](https://user-images.githubusercontent.com/1937582/190605746-126ce8b0-370c-400e-ab54-9885408e9e3b.png)

It's really hard to see in the second image here, but the corner is jagged.